src/runtime/pkg/katautils/config.go

```diff
@@ -1009,15 +1009,16 @@ func newClhHypervisorConfig(h hypervisor) (vc.HypervisorConfig, error) {
 		return vc.HypervisorConfig{}, err
 	}
 
-	if sharedFS != config.VirtioFS && sharedFS != config.VirtioFSNydus {
-		return vc.HypervisorConfig{}, errors.New("clh only support virtio-fs or virtio-fs-nydus")
-	}
-
-	if h.VirtioFSDaemon == "" {
+	if (sharedFS == config.VirtioFS || sharedFS != config.VirtioFSNydus) && h.VirtioFSDaemon == "" {
```

Comment:
Did you mean `sharedFs == config.VirtioFSNydus` here?

Reply:
I did. :-)